website/src/pages/create/assistant_reply.tsx

```diff
@@ -0,0 +1,98 @@
+import { Textarea } from "@chakra-ui/react";
+import { useRef, useState } from "react";
+import useSWRMutation from "swr/mutation";
+import useSWRImmutable from "swr/immutable";
+
+import fetcher from "src/lib/fetcher";
+import poster from "src/lib/poster";
+import { Messages } from "@/components/Messages";
```

Comment:
Can you import the local packages in a consistent way? either `src/` or `@/`?

Reply:
vscode inserted all the `@`s, probably from `jsconfig.json`.

I will reset to `src/` to consistent with other files